SparkPost/2web2ui
# AC-1971: Global tracking for all forms in billing

AC-1971: Global tracking for all forms in billing

### What Changed

- Allowlisted forms from billing page - "add-ips-form", "change-plan-form", "enable-automaticbilling-form", "billing-update-contact" , "billing-update-payment-form"
- Blocklisted sensitive fields from the form
- Added input hidden fields for CurrentPlan Section and Selected Plan section 
- Added cypress tests covering these forms

### How To Test

- Check if "add-ips-form", "change-plan-form", "enable-automaticbilling-form", "billing-update-contact" , "billing-update-payment-form" send correct submission data to segment and have blocklisted the sensitive fields

### To Do
